google-cloud-kms/google-cloud-kms.gemspec

```diff
@@ -3,7 +3,7 @@
 
 Gem::Specification.new do |gem|
   gem.name          = "google-cloud-kms"
-  gem.version       = "0.3.0"
+  gem.version       = "1.0.0"
```

Comment:
hooray for `1.x.x` releases :tada:; [semver](https://semver.org/) is poorly defined for pre-major releases ... and I *love* semver.